dspace-api/src/main/java/org/dspace/scripts/ProcessServiceImpl.java

```diff
@@ -72,7 +73,34 @@ public class ProcessServiceImpl implements ProcessService {
     private MetadataFieldService metadataFieldService;
 
     @Autowired
-    private EPersonService ePersonService;
+    private ConfigurationService configurationService;
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
```

Comment:
In today's Developer meeting, I mentioned that it'd be great to try to find a way to add automated tests for the logic in this `afterPropertiesSet()` method.  But, I'm now realizing that would be very complex, because this code triggers as soon as the ProcessService loads/initializes.

So, this comment is to just to say, we may need to test this behavior manually by starting a long-running process, and then immediately shutting down Tomcat.  Then boot back up and make sure that process has failed an its logs are attached.